### DISCUSSION TITLE
Windows CMD AIO cannot download container. Incorrect username or password???

### DISCUSSION THREAD

**Discussion by memece3** [nextcloud/all-in-one#5056] [Questions] [answered]
C:\Windows\System32>docker run ^
More? --init ^
More? --sig-proxy=false ^
More? --name nextcloud-aio-mastercontainer ^
More? --restart always ^
More? --publish 80:80 ^
More? --publish 8080:8080 ^
More? --publish 8443:8443 ^
More? --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config ^
More? --volume //var/run/docker.sock:/var/run/docker.sock:ro ^
More? nextcloud/all-in-one:latest
Unable to find image 'nextcloud/all-in-one:latest' locally
docker: Error response from daemon: Head "https://registry-1.docker.io/v2/nextcloud/all-in-one/manifests/latest": unauthorized: incorrect username or password.
See 'docker run --help'.

C:\Windows\System32>

Can someone please explain why this is happening?

**Comment by memece3** [comment]
I had this installed perfectly yesterday and the passphrase would not let me login.
So I uninstalled WS2, Docker Desktop and restarted the PC. Reinstalled Docker Desktop and WS2, and went to CMD and entered the commands just like in the instructions and just like I did before.
Did I not delete something? Is it finding some old config or something or is something broke on Github??

**Comment by szaimen** [comment, marked as answer]
> Unable to find image 'nextcloud/all-in-one:latest' locally
> docker: Error response from daemon: Head "https://registry-1.docker.io/v2/nextcloud/all-in-one/manifests/latest": unauthorized: incorrect username or password.

Hi, I think this is unrelated to AIO. it simply looks like it cannot reach registry-1.docker.io. Probably best to open a new thread in a docker forum in order to get help on this.
